deeplabcut/generate_training_dataset/trainingsetmanipulation.py

```diff
@@ -728,6 +729,7 @@ def create_training_dataset(
     net_type=None,
     augmenter_type=None,
     posecfg_template=None,
+    superanimal_name = ""
```

Comment:
Is this argument meant to be user-facing? Or only for internal purposes? It is very redundant with posecfg_template.

Reply:
I am not sure it was me who put it there. Is it used anywhere in the source code?